homeassistant/components/systemair/sensor.py

```diff
@@ -0,0 +1,505 @@
+        SENSOR_CURRENT_FAN_MODE,
+        SENSOR_USER_LOCK,
+        SENSOR_FUNCTION_COOLING,
+        SENSOR_FUNCTION_FREE_COOLING,
+        SENSOR_FUNCTION_HEATING,
+        SENSOR_FUNCTION_DEFROSTING,
+        SENSOR_FUNCTION_HEAT_RECOVERY,
+        SENSOR_FUNCTION_COOLING_RECOVERY,
+        SENSOR_FUNCTION_MOISTURE_TRANSFER,
+        SENSOR_FUNCTION_SECONDARY_AIR,
+        SENSOR_FUNCTION_COOKER_HOOD,
+        SENSOR_FUNCTION_HEATER_COOLDOWN,
+        SENSOR_FUNCTION_SERVICE_USER_LOCK,
+        SENSOR_ALARM_INCORRECT_MANUAL_MODE,
+        SENSOR_ALARM_RH,
+        SENSOR_ALARM_CO2,
+        SENSOR_ALARM_REHEATER_OVERHEAT,
+        SENSOR_ALARM_MANUAL_STOP,
+        SENSOR_ALARM_EXTERNAL_STOP,
+        SENSOR_ALARM_EXTRA_CONTROLLER,
+        SENSOR_ALARM_CHANGE_FILTER,
+        SENSOR_ALARM_PDM_TEMP,
+        SENSOR_ALARM_EFFICIENCY_TEMP_SENSOR,
+        SENSOR_ALARM_PREHEATER_TEMP_SENSOR,
+        SENSOR_ALARM_EXTRACT_TEMP_SENSOR,
+        SENSOR_ALARM_INDOOR_TEMP_SENSOR,
+        SENSOR_ALARM_SUPPLY_TEMP_SENSOR,
+        SENSOR_ALARM_REHEATER_TEMP_SENSOR,
+        SENSOR_ALARM_OUTDOOR_TEMP_SENSOR,
+        SENSOR_ALARM_BYPASS_DAMPER_2,
+        SENSOR_ALARM_ROTARY_EXCHANGER,
+        SENSOR_ALARM_BYPASS_DAMPER,
+        SENSOR_ALARM_ELECTRICAL_HEATER,
+        SENSOR_ALARM_EXTRACT_FLOW_PRESSURE,
+        SENSOR_ALARM_SUPPLY_FLOW_PRESSURE,
+        SENSOR_ALARM_EXTRACT_RPM,
+        SENSOR_ALARM_SUPPLY_RPM,
+        SENSOR_ALARM_DEFROST,
+        SENSOR_ALARM_FROST_PROTECTION_SENSOR,
+        SENSOR_ALARM_FROST_PROTECTION,
+        SENSOR_ALARM_PDM_RH,
+        SENSOR_FUNCTION_VACUUM_CLEANER,
+        SENSOR_TARGET_TEMPERATURE
+    )
+
+    global SENSOR_TYPES
+    SENSOR_TYPES = {
+        ATTR_FAN_SPEED_EXTRACT: [
+            'Extract Fan-Speed',
+            'RPM',
+            'mdi:air-conditioner',
+            SENSOR_FAN_SPEED_EXTRACT
+
+        ],
+        ATTR_FAN_SPEED_SUPPLY: [
+            'Supply Fan-Speed',
+            'RPM',
+            'mdi:air-conditioner',
+            SENSOR_FAN_SPEED_SUPPLY
+        ],
+
+        ATTR_OUTSIDE_TEMPERATURE: [
+            'Outside Temperature',
+            TEMP_CELSIUS,
+            'mdi:thermometer',
+            SENSOR_TEMPERATURE_OUTDOOR
+        ],
+        ATTR_SUPPLY_TEMPERATURE: [
+            'Supply Temperature',
+            TEMP_CELSIUS,
+            'mdi:thermometer',
+            SENSOR_TEMPERATURE_SUPPLY
+        ],
+        ATTR_EXTRACT_TEMPERATURE: [
+            'Extract Temperature',
+            TEMP_CELSIUS,
+            'mdi:thermometer',
+            SENSOR_TEMPERATURE_EXTRACT
+        ],
+
+        ATTR_CURRENT_HUMIDITY: [
+            'Inside Humidity',
+            '%',
+            'mdi:water-percent',
+            SENSOR_CURRENT_HUMIDITY
+        ],
+
+        ATTR_FILTER_TIME: [
+            'Filter Replacement',
+            'days',
+            'mdi:alarm',
+            SENSOR_FILTER_TIME
+        ],
+
+        ATTR_CURRENT_FAN_MODE: [
```

Comment:
Isn't this already part of the climate entity? We shouldn't duplicate state info between different types of entities.